spec/dummy/spec/integration/blogs_spec.rb

```diff
@@ -40,7 +40,7 @@
     get 'retrieves a specific blog' do
       operation_description 'For the id passed in the path, it searches and retrieves a blog against that id. If blog not found it returns a 404'
       produces 'application/json'
-      parameter :id, :in => :path, :type => :string
+      parameter :id, :in => :path, :type => :string, :required => true
```

Comment:
Given that path params are always required, would be convenient to have the generator set this automatically.